test_resource/guardrail.ini

```diff
@@ -27,23 +28,25 @@ coverage_percentage = 85
 allowed_mutants_percentage = 30
```

Comment:
Should this be changed to 20? Since in the git-workflow configuration it's defined as 20?

Reply:
this ini file is just to read and validate the ini values are getting populated to program variables. This has no impact on actual mutation gate